propulsion-summary-projector/Program.fs

```diff
@@ -377,4 +377,4 @@ let run argv =
 [<EntryPoint>]
 let main argv =
     try run argv
-    finally Log.CloseAndFlush()
\ No newline at end of file
+    finally Log.CloseAndFlush()
```

Comment:
hm, need to add an .editorconfig - this repo doesn't have 'redundant' EOL at EOF